sale_pricelist_triple_discount/model/pricelist.py

```diff
@@ -1,22 +1,48 @@
 # Copyright 2019 Simone Rubino - Agile Business Group
+# Copyright (c) 2021 Andrea Cometa - Apulia Software s.r.l.
+# Copyright 2022 Alberto Re - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-import openerp.addons.decimal_precision as dp
-from openerp import fields, models
+from odoo import fields, models
 
 
 class ProductPricelistItem(models.Model):
     _inherit = "product.pricelist.item"
 
     discount2 = fields.Float(
         "Discount 2 (%)",
-        digits_compute=dp.get_precision("Discount"),
+        digits="Discount",
         help="Second discount applied on a sale order line.",
         default=0.0,
     )
 
     discount3 = fields.Float(
         "Discount 3 (%)",
-        digits_compute=dp.get_precision("Discount"),
+        digits="Discount",
         help="Third discount applied on a sale order line.",
         default=0.0,
     )
+
+    def _compute_price(self, price, price_uom, product, quantity=1.0, partner=False):
+        """Compute the unit price of a product in the context of a pricelist
+        application. The unused parameters are there to make the full
+        context available for overrides.
+        """
+        self.ensure_one()
+
+        if self.compute_price != "formula":
+            return super()._compute_price(price, price_uom, product, quantity, partner)
```

Comment:
Here and below
```suggestion
            return super()._compute_price(price, price_uom, product, quantity=quantity, partner=partner)
```
otherwise if the kwargs order is changed, this leads to bugs hard to detect.